pages/compendium/index.js

```diff
@@ -14,16 +14,38 @@ const COMPENDIUM_ITEMS = [
   "Languages",
 ];
 
+const compendiumStyle = css`
+  display: grid;
+  grid-template-columns: auto 400px;
+  grid-template-rows: 100vh;
+`;
```

Comment:
Example using String format (import from `@emotion/react`)

---

Note that since it's in JS you can make composed styles:

```js
const backgroundColor: '#fff';
const backgroundStyle = css`
  background-color: ${backgroundColor};
`;

const myPageStyle = css`
  ${backgroundStyle}
  padding: 20px;
`;